epoxy-processor/src/main/java/com/airbnb/epoxy/PoetExtensions.kt

```diff
@@ -78,6 +79,10 @@ private fun JavaClassName.getPackageNameInKotlin(): String {
     return packageName()
 }
 
+fun isLambda(type: JavaTypeName): Boolean {
```

Comment:
seems that this should check the kotlin package too, since any other random class could technically contain this

Reply:
Added a check to see if it also contains the keyword `kotlin`.